myuw/templates/base.html

```diff
@@ -150,6 +151,24 @@ <h2 class="sr-only" id="app_messages_desktop_label">MyUW Announcements</h2>
         <!-- end message banner -->
         {% endblock %}
 
+	{% if display_onboard_message %}
+	{% block desktop_tour_banner %}
+        <!-- NTS: message banner based off the banner above (Desktop only) -->
+        <div class="myuw-banner-desktop app-messages" id="tour_messages_desktop" aria-labelledby="app_messages_desktop_label" role="complementary">
+            <div class="message-banner">
+                <h2 class="sr-only" id="app_messages_desktop_label">MyUW Announcements</h2>
```

Comment:
Same issue here. This is the ID used by the desktop app messages. 

**tour_messages_desktop_label** is probably better.